### PR TITLE
Postgres

### PR DESCRIPTION
**Pull request by sujithavikram**
@zhujohnny @CareyL96 @evelyncc please review this pull request

### CODE REVIEW

File: database/db.js [hunk 1]
```diff
@@ -38,21 +36,69 @@ const UpdateCommentSchema = new mongoose.Schema({
   }],
 });
 
-const UpdateAndComment = mongoose.model('UpdatesComments', UpdateCommentSchema);
+const Updatecomment = mongoose.model('Updatecomment', UpdateCommentSchema);
```
Comment (zhujohnny) [inline database/db.js]: CamelCase for Updatecomment? I'm not sure...

File: database/postgres/postgres.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        console.error('connection error-', err);
+    } else {
+        console.log('connected');
+    }
+});
+
+
+const createDataProject = function(start, end, file) {
+    for (let i = 0; i < 5000; i++) {
+    var insertArr = [];
```
Comment (zhujohnny) [inline database/postgres/postgres.js]: const?

File: database/postgres/schema.sql [hunk 3]
```diff
@@ -0,0 +1,14 @@
+CREATE DATABASE updatessdc;
+
+CREATE TABLE PROJECT(id INTEGER PRIMARY KEY NOT NULL,createdAt timestamp);
+
```
Comment (zhujohnny) [inline database/postgres/schema.sql]: new lines could make this easier to read

File: database/db.js (hunk 1 above)
Comment (sujithavikram) [reply database/db.js]: changed to camelCase

File: database/postgres/postgres.js (hunk 2 above)
Comment (sujithavikram) [reply database/postgres/postgres.js]: I'm not using this for data generation, commenting out code

File: database/postgres/schema.sql (hunk 3 above)
Comment (sujithavikram) [reply database/postgres/schema.sql]: Sure

**Pull request merged**